vcell-core/src/main/java/cbit/vcell/mapping/NetworkTransformer.java

```diff
@@ -745,9 +745,19 @@ private void transform(SimulationContext simContext, SimulationContext transform
 		mathMappingCallback.setMessage("generating network: adding observables...");
 		mathMappingCallback.setProgressFraction(progressFractionQuota/8.0f*7.0f);
 		startTime = System.currentTimeMillis();
+
+		long removeObsTime = 0;
+		long addParamTime = 0;

```

Comment:
remove this timing code in production code (you can refer to previous commit if you want to find it again).

Reply:
done